app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt

```diff
@@ -870,6 +879,38 @@ class MeetingViewModel(
     hmsSDK.stopScreenshare(actionListener)
   }
 
+  private fun getRandomVirtualBackgroundBitmap(context: Context?) : Bitmap {
+
+    val imageList = ArrayList<String>()
+    imageList.add("2.jpeg")
+    imageList.add("mountains.jpg")
+    imageList.add("tree.jpg")
+
+    val randomIndex = Random.nextInt(imageList.size);
+    return getBitmapFromAsset(context!!.applicationContext,imageList[randomIndex])!!
+  }
+
+
+  fun startPlugin(context: Context?) {
```

Comment:
rename to `startVirtualBackgroundPlugin`

Reply:
done